app/screens/login/form.tsx

```diff
@@ -326,7 +326,7 @@ const LoginForm = ({config, extra, serverDisplayName, launchError, launchType, l
                 disableFullscreenUI={true}
                 enablesReturnKeyAutomatically={true}
                 error={error}
-                keyboardType='default'
+                keyboardType={isPasswordVisible ? 'visible-password' : 'default'}
```

Comment:
Do we need to make any distinction between iOS and Android? Or will this work fine for both?

Reply:
this `visible-password` option is android only, is that a valid enough solution?

Reply:
I was reaching out to @larkox  internally since I haven't been able to validate the same behavior in IOSs. My local dev is not working for IOS simulator. Actually I am not sure if the same error happens in IOS.

Reply:
I tested on device, and I don't see anything out of the ordinary.

Reply:
@enahum ok, and @larkox also validated the issue do not happen in IOS since it forces the use of their own keyboard in password fields, meaning that the solution in this PR is enough to solve the reported issue.